debug/index.html

```diff
@@ -52,13 +58,28 @@ <h1 class="title">Transmux Analyzer</h1>
               the player on this page.
             </legend>
             <fieldset>
-              <input id="original-active" type=radio name=active checked value="original">
-              <label>
-                Your original MP2T segment:
-                <input type="file" id="original">
-              </label>
+              <legend>M2TS Input</legend>
+              <div>
+                <input id="original-active" type=radio name=active checked value="original">
+                <label>
+                  Your original MP2T segment:
+                  <input type="file" id="original">
+                </label>
+              </div>
+              <div>
+                <label><input id="combined-output" type=checkbox name=combined checked value="combined">&nbsp;Remux output into a single output?
```

Comment:
It's only a test page. I needed a little spacing because it was too close... ::deal_with_it.gif:: :smile: